backend/src/Altinn.Authorizationadmin/Altinn.AuthorizationAdmin.Core/Helpers/PolicyHelper.cs

```diff
@@ -202,7 +208,7 @@ public static string GetAltinnAppDelegationPolicyPath(string org, string app, st
             }
             else
             {
-                return $"resourceregistry /{resourceRegistryId}/{offeredBy}/{coveredBy}/ delegationpolicy.xacml";
+                return $"resourceregistry/{resourceRegistryId.AsFileName()}/{offeredBy.AsFileName()}/{coveredByPrefix.AsFileName()}/delegationpolicy.xml";
```

Comment:
missing the actual coveredBy in path, now only sets the prefix. I'm surprised this doesn't cause everything to fail 🤔